bigchaindb/backend/localmongodb/query.py

```diff
@@ -300,11 +300,11 @@ def get_validator_set(conn, height=None):
 
 
 @register_query(LocalMongoDBConnection)
-def get_validator_set_by_election_id(conn, election_id):
+def get_result_by_election_id(conn, election_id, table):
```

Comment:
Creating a new collection for every election might be cumbersome. Instead, we can introduce an `elections` collection storing documents of the form:

```
{
    'election_id': '...',
    'type': '...',
    'height': X
}
```

Reply:
What would be the use of `'type'`?

Reply:
`type` is what is `operation` in the transaction schema. For instance, if you want to find out if there was a validator update caused by the given election ID, you need to filter by type.

Reply:
I don't think you need `type` once you have the election id, there shouldn't be two elections with same election id and yet have different `operations`

Reply:
Yes, but when you receive an election ID in the command line, you do not know its type for sure. A user may look for a validator election status providing a migration election ID by mistake.